.yamllint.yaml

```diff
@@ -3,7 +3,6 @@ extends: default
 ignore: |
   .clang-tidy
   *.param.yaml
-  *.map_publisher.yaml
```

Comment:
All ROS param files should be named `*.param.yaml`, then you don't have to ignore custom names.